deep-learning/_posts/2018-04-06-GMM-HSRS.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: review
+title:  "GMM-Based Synthetic Samples for Classification of Hyperspectral Images With Limited Training Data"
+tags:   hyperspectral, classification, remote-sensing, GMM, PCA,
+author: Charles Authier
+pdf:    https://arxiv.org/pdf/1712.04778v1
+cite:
+  authors: "AmirAbbas Davari, Erchan Aptoula, Berrin Yanikoglu, Andreas Maier, Christian Riess"
+  title:   "GMM-Based Synthetic Samples for Classification of Hyperspectral Images With Limited Training Data"
+  venue:   "Published 2017 in ArXiv"
+---
+
+### Description
+A big issue in images classification is the limitation of training data. In hyperspectral, the data have high dimensional feature vector, that part can be useful. They propose to compensate this limitation by adding synthetic samples drawn from a Gaussian mixture that is estimated on the feature space.
```

Comment:
with image classification

Reply:
image classification or segmentation?

Reply:
The 2018-04-06-GMM-HSRS.md file should not be in the deep-learning folder.  It should be in the machine-learning folder instead